src/index.js

```diff
@@ -41,18 +41,29 @@ export default class Form extends React.Component {
   floatingLabelFocusStyle = this.props.focusStyle || {};
 
   handleEmojiText = (input) => {
-    var emoji = new EmojiConvertor();
-    return emoji.replace_colons(input);
+    if(_.isString(input)) {
+      const emoji = new EmojiConvertor();
+      return emoji.replace_colons(input);
+      }
+     else return input;
   }
 
   handleChange = (field, value) => {
-    const valueWithEmoji = this.handleEmojiText(value);
+    const { fields } = this.props;
+    const wholeField = fields.find(({name, key}) => name === field || key === field)
+    
+    let valueConditionallyWithEmoji = value;
```

Comment:
The variable name contains the function itself. What if we would like to add an another transformation/modification? In this case the name should be valueConditionallyWithEmojiAndCapitalizationOrUppercaseTransormation. :)

Reply:
maybe tranformedValue will be enough informative and does not contains implementation.

Reply:
I was looking for a name that reflects that the value _could_ be transformed, but maybe isn't. Because most of the time the value is not transformed. But sure, I can change it if brevity has a higher priority here.